teach_base/lesson_02/03_array_step_5.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+# Fill the array with numbers from 10 to 100 in increments of 5
+
+array = []
+i = 10
+
+while i <= 100
+  array.push i
```

Comment:
Don't pass arguments without brackets. You can use any of these:
array.append(i)
array.push(i)
array << i
array[-1] = i
array.insert(-1, i)